charts/subgraph-radio/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v2
+name: subgraph-radio
+description: Deploy a Graphcast Subgraph Radio into your Kubernetes stack
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+# renovate: image=ghcr.io/graphops/docker-builds/subgraph-radio
+appVersion: 0.1
```

Comment:
To clarify about not expected to follow semantic versioning, is it still expected to update the version here when there is a minor version update (i.e. if we release a 0.2.0)?